starters/tailwind/src/TagGroup.tsx

```diff
@@ -48,6 +48,7 @@ const tagStyles = tv({
   },
   compoundVariants: (Object.keys(colors) as Color[]).map((color) => ({
     isSelected: false,
+    isDisabled: false,
```

Comment:
I think we should add some styles, this doesn't meet contrast in dark mode
![Screenshot 2024-10-24 at 7 46 35 am](https://github.com/user-attachments/assets/bbed4500-19c7-4051-a27f-217b969decc4)

Reply:
Since the styles weren't applied prior to this change and I didn't change anything visual, would you mind picking the colors you'd like to use for the disabled state?